packages/ignition/src/app/structured-data/article.ts

```diff
@@ -0,0 +1,16 @@
+import { CreativeWork } from ".";
```

Comment:
Do not use import like that, it creates redundant references. Next can handle that, so, it works, but it's not ok